src/NexusMods.App.UI/WorkspaceSystem/WorkspaceController/IWorkspaceController.cs

```diff
@@ -112,6 +113,11 @@ public interface IWorkspaceController
     /// </summary>
     public void SetIcon(IconValue? icon, WorkspaceId workspaceId, PanelId panelId, PanelTabId tabId);
 
+    public OpenPageBehavior GetOpenPageBehavior(
+        PageData requestedPage,
+        NavigationInformation navigationInformation,
+        Optional<PageIdBundle> currentPage);
```

Comment:
I would set a default here for currentPage:
```csharp
public OpenPageBehavior GetOpenPageBehavior(
        PageData requestedPage,
        NavigationInformation navigationInformation,
        Optional<PageIdBundle> currentPage = Optional<PageIdBundle>.None);
```
So that the users outside a page don't have to set it every time.

Reply:
No, I'd agree if we mostly call this method from outside the workspace system, but that's more an exception than the rule.